client/src/pages/Login/Login.tsx

```diff
@@ -31,30 +45,22 @@ const Login = () => {
       exit={{ opacity: 0, y: -20 }}
     >
       <FormSection>
-        <LoginForm onSubmit={(e: React.FormEvent) => handleSubmit(e)}>
+        <LoginForm onSubmit={handleSubmit(submitForm)}>
```

Comment:
This should look something along the lines of

<LoginForm onSubmit={handleSubmit(submitForm)}>

the 'submitForm function that was left blank would be where all the logic would be written for what you want the form to do once submitted.